### PR TITLE
conda, conda-create: add German translation

### PR DESCRIPTION
**Pull request by marciska**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.de/common/conda-create.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# conda create
+
+> Erstellen von neuen Conda Umgebungen.
```
Comment (marchersimon) [inline pages.de/common/conda-create.md:3]:
```suggestion
> Erstelle neue Conda-Umgebungen.
```

File: pages.de/common/conda-create.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> Mehr Informationen: <https://docs.conda.io/projects/conda/en/latest/commands/create.html>.
+
+- Erstelle eine neue Umgebung mit dem Namen `py39`, und installiere Python 3.9 und NumPy v1.11 (oder höher) in ihr:
```
Comment (marchersimon) [inline pages.de/common/conda-create.md:6]:
```suggestion
- Erstelle eine neue Umgebung mit dem Namen `py39` und installiere Python 3.9 und NumPy v1.11 (oder höher) darin:
```

File: pages.de/common/conda-create.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`conda create --yes --name {{py39}} python={{3.9}} "{{numpy>=1.11}}"`
+
+- Erstelle eine exakte Kopie einer Umgebung:
+
+`conda create --clone {{py39}} --name {{py39-copy}}`
+
+- Erstelle eine neue Umgebung mit gegebenem Namen und den zu installierenden Paketen:
+
+`conda create --name {{umgebungs_name}} {{paket_name}}`
```
Comment (marchersimon) [inline pages.de/common/conda-create.md:16]:
```suggestion
`conda create --name {{umgebungsname}} {{paketname}}`
```

File: pages.de/common/conda.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# conda
+
+> Ein Kommandozeilentool und Paketverwaltung für beliebige Programmiersprachen.
+> Manche Unterbefehle (z.B. `conda create`) besitzen ihre eigene Dokumentation.
```
Comment (marchersimon) [inline pages.de/common/conda.md:4]: We have a template for this: <https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md#de>
```suggestion
> Manche Unterbefehle wie `conda create` sind separat dokumentiert.
```

File: pages.de/common/conda-create.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# conda create
+
+> Erstellen von neuen Conda Umgebungen.
+> Mehr Informationen: <https://docs.conda.io/projects/conda/en/latest/commands/create.html>.
```
Comment (marchersimon) [inline pages.de/common/conda-create.md:4]:
```suggestion
> Weitere Informationen: <https://docs.conda.io/projects/conda/en/latest/commands/create.html>.
```

File: pages.de/common/conda.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+> Mehr Informationen: <https://github.com/conda/conda>.
```
Comment (marchersimon) [inline pages.de/common/conda.md:5]:
```suggestion
> Weitere Informationen: <https://github.com/conda/conda>.
```

File: pages.de/common/conda.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+- Erstelle eine neue Umgebung mit den zu installierenden Paketen:
+
+`conda create --name {{umgebungs_name}} {{python=3.9 matplotlib}}`
```
Comment (marchersimon) [inline pages.de/common/conda.md:9]:
```suggestion
`conda create --name {{umgebungsname}} {{python=3.9 matplotlib}}`
```

File: pages.de/common/conda.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+- Liste alle Umgebungen auf:
+
+`conda info --envs`
+
+- Eine Umgebung laden:
```
Comment (marchersimon) [inline pages.de/common/conda.md:15]:
```suggestion
- Lade eine Umgebung:
```

File: pages.de/common/conda.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+`conda {{activate umgebungs_name}}`
+
+- Eine Umgebung entladen:
```
Comment (marchersimon) [inline pages.de/common/conda.md:19]:
```suggestion
- Entlade eine Umgebung:
```

File: pages.de/common/conda.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+
+`conda {{deactivate}}`
+
+- Eine Umgebung löschen (entferne alle Pakete):
```
Comment (marchersimon) [inline pages.de/common/conda.md:23]:
```suggestion
- Lösche eine Umgebung (entferne alle Pakete):
```

File: pages.de/common/conda.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+
+`conda remove --name {{umgebungs_name}} --all`
```
Comment (marchersimon) [inline pages.de/common/conda.md:25]:
```suggestion
`conda remove --name {{umgebungsname}} --all`
```

File: pages.de/common/conda.md [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+
+- Installiere Pakete in die derzeit geladene Umgebung:
+
+`conda install {{python=3.4 numpy}}`
+
+- Auflistung aller installierten Pakete in der derzeit geladenen Umgebung:
```
Comment (marchersimon) [inline pages.de/common/conda.md:31]:
```suggestion
- Liste alle installierten Pakete in der derzeit geladenen Umgebung auf:
```

**Comment by marciska** [comment]
I agree with all proposed changes.

File: pages.de/common/conda.md [hunk 13]
```diff
@@ -0,0 +1,37 @@
+# conda
+
+> Ein Kommandozeilentool und Paketverwaltung für beliebige Programmiersprachen.
```
Comment (pixelcmtd) [inline pages.de/common/conda.md:3]:
```suggestion
> Eine Paket-, Abhängigkeits- und Umgebungsverwaltung für beliebige Programmiersprachen.
```

**Comment by marciska** [comment]
(Hopefully) last review has been resolved.

**Review by pixelcmtd — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @marciska

**Pull request merged**